website/public/deploymentguide copy/index.html

```diff
@@ -0,0 +1,579 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0"
+    />
+    <link rel="stylesheet" href="/css/index.css" />
+    <link rel="icon" type="image/png" href="/favicon.png" />
+    <script type="module" src="/js/index.js"></script>
+    <title>CiviForm | Deployment Guide</title>
```

Comment:
**suggestion:** I'd stray from calling this a deployment guide, since I feel like that may get misunderstood (my initial thought was that it would go through AWS setup, etc.). Maybe "Preparation Guide" or something would be better? Could you also update the folder name and make it consistent with the other formats (i.e. preparation-guide

Reply:
How about "implementation guide"?

Reply:
Thanks! I will make sure that we have the right one.